controller/rancher/rancher.go

```diff
@@ -310,6 +347,7 @@ func (lbc *LoadBalancerController) BuildConfigFromMetadata(lbName, envUUID, self
 		StickinessPolicy: &lbMeta.StickinessPolicy,
 	}
 
+	//Q: What does ProcessCustomConfig do?
```

Comment:
@prachidamle to answer the question - it merges custom and default config of haproxy. Should not affect functionality you are adding.